repository/src/main/java/org/apache/atlas/repository/graph/GraphHelper.java

```diff
@@ -376,6 +376,30 @@ public static AtlasVertex getClassificationVertex(AtlasVertex entityVertex, Stri
         return ret;
     }
 
+    public static List<AtlasVertex> getAllClassificationVertex(AtlasGraph graph, String classificationName) {
```

Comment:
Rename to "getClassificationVerticesByTypeName"

Reply:
Do you think we should add pagination logic here?

Reply:
Normally edge deletion process removes all classificationVertices so this is just in case anything remains. So it shouldn't be more than 100/1000